docs/guides/billing.md

```diff
@@ -35,7 +41,7 @@ This setting is a collection of billing plans. Each billing plan allows the foll
 | -------------- | ---------- | :-------: | :-----------: | ---------------------------------------------------------- |
 | `interval`     | `ONE_TIME` |    Yes    |       -       | `BillingInterval.OneTime` value                            |
 | `amount`       | `number`   |    Yes    |       -       | The amount to charge                                       |
-| `currencyCode` | `string`   |    Yes    |       -       | The currency to charge, currently only `"USD"` is accepted |
+| `currencyCode` | `string`   |    Yes    |       -       | The currency to charge, USD or merchant's shop currency |
```

Comment:
This changed as of `2023-04`
https://shopify.dev/changelog/graphql-admin-api-now-supports-creating-app-charges-using-the-merchant-s-billing-currency